lms/static/sass/xmodule/_headings.scss

```diff
@@ -88,29 +88,33 @@ $headings-base-color:                       $gray-d2;
     letter-spacing: 1px;
 }
 
-.xblock .xblock {
+// let's make these classnames available to the entire LMS!
+
+.hd-1,
```

Comment:
I could. Do you think it's more helpful to have them in their own file though? Since these are duplicates of whatever we have set in the UXPL, if the UXPL changes, we'd need to update these accordingly and I feel like having them in their own file is easier to manage. What do you think?